charts/opensearch-dashboards/values.yaml

```diff
@@ -199,7 +199,9 @@ autoscaling:
   enabled: false
   minReplicas: 1
   maxReplicas: 10
-  targetCPUUtilizationPercentage: 80
+  # The legacy `targetCPUUtilizationPercentage` key has been deprecated in favor of `targetCPU`
+  targetCPU: "80"
```

Comment:
@Simple-Analysis can we also add a 1 liner description the [README](https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch/README.md) which will help users better use this feature?

Reply:
> @Simple-Analysis can we also add a 1 liner description the [README](https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch/README.md) which will help users better use this feature?

Definitely. I added descriptions for the autoscaling parameters.